common/types.go

```diff
@@ -101,6 +103,7 @@ type ReadParams struct {
 	// The name of the object we are reading, e.g. "Account"
 	ObjectName string // required
 	// The fields we are reading from the object, e.g. ["Id", "Name", "BillingCity"]
+	// NOTE: to access this value use ReadParams.GetUniqueFields.
 	Fields []string // required, at least one field needed
```

Comment:
Can we make this a string set? That way, we don't have to remember the unique function on top of it

Reply:
I prefer that too. I wasn't sure about usage of `handy.Set` outside for those who will use this function.